config.go

```diff
@@ -536,13 +530,22 @@ func LoadConfig(path string) (*Config, error) {
 			}
 		case "outputs":
 			for outputName, outputVal := range subtbl.Fields {
-				outputSubtbl, ok := outputVal.(*ast.Table)
-				if !ok {
-					return nil, err
-				}
-				err = c.parseOutput(outputName, outputSubtbl)
-				if err != nil {
-					return nil, err
+				switch outputSubtbl := outputVal.(type) {
+				case *ast.Table:
+					err = c.parseOutput(outputName, outputSubtbl, 0)
+					if err != nil {
+						return nil, err
```

Comment:
Maybe we add some (ideally stderr) printf output here saying that outputName failed to parse? Would be good to catch that in the logs.

```
fmt.Printf("Could not parse config for output: %s\n", outputName)
```